jormungandr/Cargo.toml

```diff
@@ -55,12 +55,13 @@ slog-term = "2.6.0"
 structopt = "^0.3"
 thiserror = "1.0.21"
 tokio = { version = "^1.0", features = ["rt-multi-thread", "time", "sync", "rt", "signal", "test-util"] }
-tokio-stream = "0.1.0"
+tokio-stream = "0.1.2"
 tokio-util = { version = "0.6.0", features = ["time"] }
-tonic = "0.3"
+tonic = "0.4"
 async-trait = "0.1"
 lru = "^0.6.1"
-warp = { version = "0.2.4", features = ["tls"] }
+warp = { version = "0.3", features = ["tls"] }
+tower = "0.4.4"
```

Comment:
Is the explicity `tower` dependency intended?

Reply:
No, it is a leftover indeed.